test-fixtures/monorepo/case3-shared-dependencies2/package-c/spago.yaml

```diff
@@ -7,4 +7,3 @@ package:
     dependencies:
       - console
       - effect
-      - prelude
```

Comment:
Why remove these? They are testing a specific scenario (more info in the test code)

Reply:
But these are _test_ dependencies. The comment on the test suggests it's testing the core dependencies, and I didn't touch those.

The reason I removed these is that I wanted to make it also test how the transitive test dependencies are computed.

Reply:
Sounds good 👍